### PR TITLE
Descriptions for chain-config

### PR DESCRIPTION
**Pull request by leeyr338**
Add:
- `--enable_version`
- `--stdout`

Fix:
- `--enable_tls`

### CODE REVIEW

File: docs/cita/configuration/chain-config.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ usage: create_cita_config.py create [-h]
                                     [--resource_dir RESOURCE_DIR]
                                     [--grpc_port GRPC_PORT]
                                     [--jsonrpc_port JSONRPC_PORT]
-                                    [--ws_port WS_PORT]
-                                    [--enable_tls]
+                                    [--ws_port WS_PORT] [--enable_tls]
```
Comment (kaikai1024) [inline docs/cita/configuration/chain-config.md:32]: One line?

Comment (leeyr338) [reply docs/cita/configuration/chain-config.md:32]: Run the command:
```shell
./bin/cita create --help
```
and will get the output like this:
```
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT] [--enable_tls]
                                    [--enable_version] [--stdout]

optional arguments:
  -h, --help            show this help message and exit
  --authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]
                        Authorities (addresses) list.
  --chain_name CHAIN_NAME
                        Name of the new chain.
  --nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]
                        Node network addresses for new nodes.
  --super_admin SUPER_ADMIN
                        Address of super admin.
  --contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]
                        Update constructor arguments for system contract. Can
                        be specify more than once.
  --timestamp TIMESTAMP
                        Specify a timestamp to use.
  --resource_dir RESOURCE_DIR
                        Chain resource directory.
  --grpc_port GRPC_PORT
                        grpc port for this chain
  --jsonrpc_port JSONRPC_PORT
                        jsonrpc port for this chain
  --ws_port WS_PORT     websocket port for this chain
  --enable_tls          The data is encrypted and transmitted on the network
  --enable_version      Jsonrpc will return cita version
  --stdout              Logs will output to stdout
```

Comment (kaikai1024) [reply docs/cita/configuration/chain-config.md:32]: Need to fix the script of CITA. @leeyr338  you can fix the CITA first.

should be:

```
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT]
                                    [--enable_tls]
                                    [--enable_version]
                                    [--stdout]
```

Comment (leeyr338) [reply docs/cita/configuration/chain-config.md:32]: Okay!

File: docs/cita/rpc-guide/rpc.md [hunk 2]
```diff
@@ -325,7 +325,7 @@ Result:
 
 ### getVersion
 
-获取当前 CITA 软件的版本号
+获取当前 CITA 软件的版本号, 该接口需要在链创建时使能后才可以使用。详细查看 [链级配置](../configuration/chain-config) 中的 `--enable_version` 选项说明。
```
Comment (boundless-forest) [inline docs/cita/rpc-guide/rpc.md]: `该接口需要在链创建时使能后才可以使用` 不太通顺吧。

Comment (leeyr338) [reply docs/cita/rpc-guide/rpc.md]: 改成  **该接口设置了使能开关，需要在链创建时通过使能选项开启该功能，才能正常使用。** 怎么样？

Comment (boundless-forest) [reply docs/cita/rpc-guide/rpc.md]: :v:

File: docs/cita/configuration/chain-config.md (hunk 1 above)
Comment (leeyr338) [reply docs/cita/configuration/chain-config.md:32]: This format is default by `argparse` library, I cannot find an easy way to get the format like:
 
```
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT]
                                    [--enable_tls]
                                    [--enable_version]
                                    [--stdout]
```

By the way, the actual result of v0.22.0 is:

```
./scripts/create_cita_config.py create -h
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT] [--enable_tls]

```

Comment (kaikai1024) [reply docs/cita/configuration/chain-config.md:32]: I see. leave it.

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**